deploy/wip-catalog/templates/apiserver.yaml

```diff
@@ -37,11 +37,18 @@ metadata:
   name: apiserver
 spec:
 {{ if .Values.debug }}
+{{ if .Values.nodePort }}
+  type: NodePort
+{{ else }}
   type: LoadBalancer
 {{ end }}
+{{ end }}
```

Comment:
@krancour I want a stable node port so that I can maintain a kubeconfig against our API server without changing ports.  If I use a load balancer service, I don't think I get that.

Separately, I'm fine refactoring how the service type is set in the chart.